tools/python/boutiques/publisher.py

```diff
@@ -4,6 +4,7 @@
 from boutiques.logger import raise_error, print_info
 from boutiques.zenodoHelper import ZenodoError, ZenodoHelper
 import json
```

Comment:
Can we remove `import json`?

Reply:
Replaces all usage of json package with simplejson